src/gui/editors/PianoRoll.cpp

```diff
@@ -2553,6 +2527,13 @@ void PianoRoll::mouseMoveEvent( QMouseEvent * me )
 				}
 			}
 		}
+		else if (me->buttons() == Qt::NoButton &&
+			( m_editMode == ModeErase ||
+			m_editMode == ModeSelect ||
+			m_editMode == ModeEditDetuning))
```

Comment:
I'm fine with this, but I'm curious why you didn't wrote it `m_editMode != ModeDraw`.

Reply:
No reason, I'll change it for simplicity :+1: